tests/lib/wifi.py

```diff
@@ -219,3 +219,19 @@ def prepare(self):
         pass
     def cleanup(self):
         pass
+
+class wifi_client_stub():
+    def disable_and_enable_wifi(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_scan(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_connect(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_connectivity_verify(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_disconnect(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_change_region(self, *args, **kwargs):
+        raise Exception("Not implemented!")
+    def wifi_ssid_disconnect(self, *args, **kwargs):
```

Comment:
Why args and kwargs? These need to be fixed. Also it might be nice the leave an example return after the exception to standardize the format of the return values

E.g.  wifi_scan returns a list of dicts with the SSID name and other info retrived from the scan.

Reply:
I followed the stub format which we have already used in wifi_stub(). Will work on this.
Am still not getting the example for return to be set after exception. Please help me on this

Reply:
I think for wifi_change_region we should suggest default arguments... maybe nothing on the latter one. Not sure why we needed these for the other stub.

Reply:
I have removed the args and kwargs, but still should I be adding the arguments used in the function?

Also I couldn't get the "example return after exception" which you have mentioned. Could you provide an example for the same and give me more clarification on this?

Reply:
yea, so this is a stub, so it should be the basis for providing a reasonable first step to implementing the feature. so know what return values are expected to be would be nice.
```
def wifi_scan(self):
     '''Function that scans for SSIDs on a particular radio, takes no arguments and
        returns a dict of ssids with a dict of attributes for an ssid (e.g. channel, snr, etc)'''
     raise Exception(...)

     # this code does not execute, but rather serves as an example for the API
     return {ssid1: { "attr1": "val1", "attr2": "val2" },
                 ssid2: {  "attr1": "val1", "attr2": "val2" } }
```

Reply:
Thanks a lot!! Got the point.. Will incorporate the same